src/extensions/scratch3_music/index.js

```diff
@@ -722,6 +778,26 @@ class Scratch3MusicBlocks {
                         }
                     }
                 },
+                {
+                    opcode: 'midiPlayDrumForBeats',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'music.midiPlayDrumForBeats',
+                        default: 'play drum [DRUM] for [BEATS] beats',
```

Comment:
Just a thought that came to mind - should there be some more obvious way to distinguish between the two "play drum" blocks? Presently the two blocks look totally identical if their DRUM slots are filled (because the only way to tell the difference is to see if it's a dropdown or a number input). Not really a big deal since this is an obsolete block, though.